docs/particles/particle_class.rst

```diff
@@ -1,18 +1,19 @@
 .. _particle-class:
 
-|Particle| Class
+Particle objects
 ****************
 
-The |Particle| class provides an object-oriented interface to access and
-represent particle information.
+PlasmaPy contains several classes to represent particles, including
+|Particle|, |CustomParticle|, |ParticleList|, and
+|DimensionlessParticle|.
 
 .. _particle-class-instantiation:
 
-Creating a |Particle| object
-============================
+Particles
+=========
 
-The simplest way to create a |Particle| object is to pass it a `str`
-representing a particle.
+To create a |Particle| object, pass it a |particle-like| string that
+represents a particle.
```

Comment:
This links it to the glossary term for particle-like, which goes into a lot of detail.